server/src/main/java/org/jboss/as/server/ServerEnvironment.java

```diff
@@ -121,37 +121,6 @@ public ProcessType getProcessType() {
      */
     public static final String HOME_DIR = "jboss.home.dir";
 
-    /**
-     * Constant that holds the name of the system property for specifying the directory returned from
-     * {@link #getModulesDir()}.
-     *
-     * <p>
-     * Defaults to <tt><em>HOME_DIR</em>/modules</tt>/
-     * </p>
-     *
-     * <strong>This system property has no real meaning and should not be regarded as providing any sort of useful
-     * information.</strong> The "modules" directory is the default location from which JBoss Modules looks to find
-     * modules. However, this behavior is in no way controlled by this system property, nor is it guaranteed that
-     * modules will be loaded from only one directory, nor is it guaranteed that the "modules" directory will be one
-     * of the directories used. Finally, the structure and contents of any directories from which JBoss Modules loads
-     * resources is not something available from this class. Users wishing to interact with the modular classloading
-     * system should use the APIs provided by JBoss Modules
-     *
-     *
-     * @deprecated has no useful meaning
-     */
-    @Deprecated
-    public static final String MODULES_DIR = "jboss.modules.dir";
-
-    /**
-     * Constant that holds the name of the system property for specifying
-     * {@link #getBundlesDir() the bundles directory}.
-     *
-     * <p>
-     * Defaults to <tt><em>HOME_DIR</em>/bundles</tt>/
-     */
-    public static final String BUNDLES_DIR = "jboss.bundles.dir";
```

Comment:
I understand we can remove this without the previous deprecation because this directory for OSGi bundles was used only internally, is that correct?

Reply:
That said, the constant itself is internal API. The server module is private.